apps/dapp/src/components/Pages/Core/NewUI/HomeInput.tsx

```diff
@@ -1,25 +1,32 @@
-import React, { InputHTMLAttributes, KeyboardEvent } from 'react';
+import { Popover } from 'components/Popover';
+import { TICKER_SYMBOL } from 'enums/ticker-symbol';
+import { useWallet } from 'providers/WalletProvider';
+import React, { InputHTMLAttributes, KeyboardEvent, useState } from 'react';
 import styled, { css } from 'styled-components';
-import { phoneAndAbove, tabletAndAbove } from 'styles/breakpoints';
 import { theme } from 'styles/theme';
-import { InputSelect, Option, SelectTempleDaoOptions } from '../../../InputSelect/InputSelect';
+import { formatToken } from 'utils/formatter';
+import ethImg from 'assets/images/newui-images/tokens/eth.png';
+import wethImg from 'assets/images/newui-images/tokens/weth.png';
+import daiImg from 'assets/images/newui-images/tokens/dai.png';
+import fraxImg from 'assets/images/newui-images/tokens/frax.png';
+import usdcImg from 'assets/images/newui-images/tokens/usdc.png';
+import usdtImg from 'assets/images/newui-images/tokens/usdt.png';
+import templeImg from 'assets/images/newui-images/tokens/temple.png';
 
-import divider from 'assets/images/divider.svg';
-import { pixelsToRems } from 'styles/mixins';
 interface SizeProps {
   small?: boolean;
 }
 
 export interface CryptoSelector {
   kind: 'select';
   // A selector for the crypto, must provide onCryptoChange
-  cryptoOptions: SelectTempleDaoOptions;
-  defaultValue?: Option;
+  cryptoOptions: TICKER_SYMBOL[];
   // use to limit the number of elements shown in the selector at anytime
   maxSelectorItems?: number;
+  selected: TICKER_SYMBOL;
```

Comment:
No longer relying on the InputSelect component, so we're passing the currently selected token too